### PR TITLE
[BrowserKit] Uppercase the "GET" method in redirects

### PR DESCRIPTION
**Pull request by jakzal**
| Q | A |
| --- | --- |
| Branch? | 2.7 |
| Bug fix? | yes |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | #19419 |
| License | MIT |
| Doc PR | - |

### CODE REVIEW

File: src/Symfony/Component/BrowserKit/Client.php [hunk 1]
```diff
@@ -464,7 +464,7 @@ public function followRedirect()
             $content = $request->getContent();
         }
 
-        if ('get' === strtolower($method)) {
+        if ('GET' === strtoupper($method)) {
```
Comment (jakzal) [inline src/Symfony/Component/BrowserKit/Client.php]: This is only changed for consistency.

**Comment by javiereguiluz** [comment]
👍 

Status: reviewed

**Comment by nicolas-grekas** [comment]
Thank you @jakzal.

**Comment by stof** [comment]
Thank you @jakzal.

**Pull request merged**